config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -229,79 +213,49 @@ private void setFailResult(ConfigFuture configFuture) {
     }
 
     /**
-     * The type Config change runnable.
+     * The type FileListener.
      */
-    class ConfigChangeRunnable implements Runnable {
-
-        private String dataId;
-        private ConfigChangeListener listener;
+    class FileListener implements ConfigurationChangeListener {
 
-        /**
-         * Instantiates a new Config change runnable.
-         */
-        public ConfigChangeRunnable() {
-        }
+        private final String dataId;
+        private final ConfigurationChangeListener listener;
+        private final ExecutorService executor = new ThreadPoolExecutor(CORE_LISTENER_THREAD, MAX_LISTENER_THREAD, 0L,
```

Comment:
Generate too many instances of executor and Thread.

Reply:
This file subscription does not have a subscription function itself. It requires a while to monopolize a thread. For each subscription, a thread pool with a single thread is required.